lib/animina/accounts/resources/fast_user.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Animina.Accounts.FastUser do
+  @moduledoc """
+  This is the Fast User module.
+  """
+
+  use Ash.Resource,
+    data_layer: AshPostgres.DataLayer,
+    authorizers: Ash.Policy.Authorizer,
+    domain: Animina.Accounts,
+    extensions: []
+
+  postgres do
```

Comment:
Is it possible to set this read only? If yes we should.

Reply:
I couldn't find any documentation on how to make a resource read only. FastUser only supports the read action